### ISSUE TITLE
hypopigmented macules

### ISSUE THREAD

**Issue by TonyRoscioli** [obophenotype/human-phenotype-ontology#7450]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
That is [Hypopigmented macule HP:0020073](https://hpo.jax.org/app/browse/term/HP:0020073)!

**Comment by TonyRoscioli** [comment]
Thanks - the search engine did not find it in the plural !

Sent from my iPhone

On 13 Apr 2022, at 9:07 pm, Peter Robinson ***@***.***> wrote:

﻿

That is Hypopigmented macule HP:0020073<https://hpo.jax.org/app/browse/term/HP:0020073>!
